stories/Components/UIcomponents/Hero/PageHero/page-hero.scss

```diff
@@ -19,6 +19,10 @@
 
     .overlay-grey {
       @extend %absolute__content;
+
+      &::before {
+        background: rgba(0, 0, 0, 0.025);
```

Comment:
@paboden The alpha value here should be `0.2` rather than `0.025`. But is there a reason we need this component-specific definition at all?

Reply:
After the cleanup, there is no reason for this css snippet. Removed.